### PR TITLE
🚀Release: Field updates, docs & coding standards updates

### PR DESCRIPTION
**Pull request by robertdevore**
This merge has a variety of updates, mainly based on code work I've been doing with it for WP Dispensary's upcoming [#WPD2](https://twitter.com/search?q=%23WPD2) release.

*sidenote: in the future, PR's will be organized into a single type of update for each pull request*

📦 **NEW:**

- title field
- separator field
- placeholder arg for text field

📖 **DOC:**

- description & authors in plugin details
- various README.md updates
- inline field names in `wposa-init.php`

👌 **IMPROVE:**

- WordPress Coding Standards FTW

💻 **CODE:**

Here are examples of the new options that come with this PR :+1: 

**Field:** type `separator`

```
// Field: Separator.
$wposa_obj->add_field(
	'wposa_basic',
	array(
		'id'      => 'separator',
		'type'    => 'separator',
	)
);
```

**Field:** type `title`

```
// Field: Title.
$wposa_obj->add_field(
	'wposa_basic',
	array(
		'id'      => 'title',
		'type'    => 'title',
		'name'    => '<h1>Title</h1>',
	)
);
```

**Field:** type `text`
**Field:** arg `placeholder`

```
// Field: Text.
$wposa_obj->add_field(
	'wposa_basic',
	array(
		'id'          => 'text',
		'type'        => 'text',
		'name'        => __( 'Text Input', 'WPOSA' ),
		'desc'        => __( 'Text input description', 'WPOSA' ),
		'default'     => 'Default Text',
		'placeholder' => '',
	)
);
```

📷 **SCREENSHOTS:**

title-and-separator-screenshot

Here, we can see the new `title` field in use, along with the `separator` field.

### CODE REVIEW

**Comment by ahmadawais** [comment]
Needs better title and description with how you teated it and screenshots.

**Comment by robertdevore** [comment]
Updated with new details :+1:

**Comment by ahmadawais** [comment]
@deviodigital that's what we call a good PR summary. I am going to make sure I share it on Twitter on something. Glad contributing to open source projects together is helping you get better. 

What's say we make a minor release of off this? Just add the release git tag and bump a version and I'll merge this PR.

Thank You! 😇

Peace! ✌️

File: wposa-init.php [hunk 1]
```diff
@@ -139,6 +147,31 @@
...
+	// Field: Separator.
+	$wposa_obj->add_field(
+		'wposa_basic',
+		array(
+			'id'      => 'separator',
+			'type'    => 'separator',
+		)
+	);
+
+
```
Comment (ahmadawais) [inline wposa-init.php:160]: Can we keep the spacing consistent? I like just one single line of space — if the entire project follows that, it looks good. Thoughts? @deviodigital

Comment (robertdevore) [reply wposa-init.php:160]: I agree. :+1: Will get that fixed up in the next round of WordPress Coding Standards updates.

**Comment by ahmadawais** [comment]
No worries I already did fix it. Make sure you fetch the lastest update

**Pull request merged**